tests/Directory.Build.props

```diff
@@ -14,11 +14,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="xunit.core" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-    <PackageReference Include="XunitXml.TestLogger" Version="2.0.0" />
-    <PackageReference Include="FluentAssertions" Version="4.19.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />

```

Comment:
There are actually some newer versions available (up to 16.4.0), but not all are compatible with all our TFMs. This is the highest common version. Would you rather fork by TFM?

Reply:
No, it's fine, I think.
At some point we're going to have to drop older TFMs, if even Microsoft doesn't support them...